Classes/common/CDTIncrementalStore.h

```diff
@@ -129,4 +59,34 @@ typedef NS_ENUM(NSInteger, CDTIncrementalStoreErrors) {
     CDTISErrorNotSupported
 };
 
+/**
+ *  The databaseName is exposed in order to be able to identify the different
+ *  CDTIncrementalStore objects. @see +storesFromCoordinator:coordinator
+ */
+@property (nonatomic, strong) NSString *databaseName;
+
+/**
+ * Create a CDTReplicator object set up to replicate changes from the
+ * local datastore to a remote database.
+ *
+ *  @param remoteURL the remote server URL to which the data is replicated.
+ *  @param error     report error information
+ *
+ *  @return a CDTReplicator instance which can be used to start and
+ *  stop the replication itself, or `nil` on error.
+ */
+- (CDTReplicator *)replicatorThatPushesToURL:(NSURL *)remoteURL withError:(NSError **)error;
+
+/**
+ * Create a CDTReplicator object set up to replicate changes from a remote database to the
+ * local datastore.
+ *
+ *  @param remoteURL the remote server URL to which the data is replicated.
+ *  @param error     report error information
+ *
+ *  @return a CDTReplicator instance which can be used to start and
+ *  stop the replication itself, or `nil` on error.
+*/
+- (CDTReplicator *)replicatorThatPullsFromURL:(NSURL *)remoteURL withError:(NSError **)error;
```

Comment:
This is covered in `## Accessing your store object`.
Having said that.. the discussion on `-linkReplicators:` needs to go.